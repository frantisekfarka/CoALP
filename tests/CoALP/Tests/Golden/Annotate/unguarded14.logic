q(s(Q1),Q2):-p(Q1,Q2).
p(t(P1),P2):-r(P1,P2).
r(u(U1),U2):-v(U1,U2).
v(w(V1),V2):-q(V2,V2).