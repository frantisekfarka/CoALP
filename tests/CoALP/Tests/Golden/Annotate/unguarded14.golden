q(s(V_2), V_3, transform-func-1(V_-10)) :- p(V_2, V_3, V_-10).
p(t(V_4), V_5, transform-func-2(V_-11)) :- r(V_4, V_5, V_-11).
r(u(V_6), V_7, transform-func-3(V_-12)) :- v(V_6, V_7, V_-12).
v(w(V_8), V_9, transform-func-4(V_-13)) :- q(V_9, V_9, V_-13).
